src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests.cs

```diff
@@ -10546,6 +10552,101 @@ class C3 { }
                 Diagnostic(ErrorCode.ERR_BadTypeArgument, "TypedReference").WithArguments("System.TypedReference").WithLocation(11, 7));
         }
 
+        [Fact]
+        public void GenericAttributeRestrictedTypeArgument_B()
```

Comment:
>B [](http://example.com/codeflow?start=59&length=1)

Consider using more informative labels for test names, such as `_UnsafeContext`

Reply:
done.